packages/agent/src/index.ts

```diff
@@ -1,3 +1,13 @@
+import 'react-native-get-random-values'
+import 'fast-text-encoding'
+
+import { Buffer } from '@credo-ts/core'
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+// eslint-disable-next-line no-undef
+global.Buffer = Buffer
```

Comment:
Why is this needed?

Reply:
We need it for cheqd support. It needs global buffer. I've added a comment